.travis.yml

```diff
@@ -2,16 +2,19 @@ language: php
 sudo: required
 
 php:
-  - 7.0
-  - 7.1
   - 7.2
   - 7.3
+  - 7.4
 
 services:
-  - postgresql
+  - docker
 
 install:
   - composer install
 
+before_script:
+  - docker-compose -f docker/docker-compose.yml up -d
+  - sh docker/waitForPostgres.sh
```

Comment:
Changes LGTM, perhaps also add some instructions to the README.md for contributors who want to run the tests locally?

Reply:
Added.